bugbug/bug_features.py

```diff
@@ -183,6 +183,16 @@ def __call__(self, bug):
         return bug['priority']
 
 
+class comment_count(object):
+    def __call__(self, bug):
+        return field(bug, 'comment_count')
+
+
+class comment_length(object):
+    def __call__(self, bug):
+        return sum([len(x['text']) for x in bug['comments']])
```

Comment:
Nit: The argument to `sum` doesn't need to be a list, it can be an iterator.
If you use a list, you'll needlessly spend time creating it. You can do `sum(len(c['text']) for c in bug['comments'])` instead.

Reply:
Got it! @marco-c 
Thank you for your guidance. Appreciate it.